packages/x-data-grid-generator/src/renderer/renderEditRating.tsx

```diff
@@ -15,7 +15,11 @@ function EditRating(props: GridRenderEditCellParams<any, number>) {
 
   const handleRef = (element: HTMLElement | undefined) => {
     if (element) {
-      element.querySelector<HTMLElement>(`input[value="${value}"]`)!.focus();
+      if (value !== 0) {
+        element.querySelector<HTMLElement>(`input[value="${value}"]`)!.focus();
+      } else {
+        element.querySelector<HTMLElement>(`input[value=""]`)!.focus();
+      }
```

Comment:
when ever user goes one up on `5` and down on `1`, `value` will be `0`. since there are no inputs with `value="0"` demos were crashing. So focusing on input with `no value (value='')`  whenever `value=0` seems to fix the issue.